frontend/src/scenes/onboarding/home/home.tsx

```diff
@@ -0,0 +1,174 @@
+import { PageHeader } from 'lib/components/PageHeader'
+import React, { useState } from 'react'
+import { PostHogLessons } from './modules/posthogLessons'
+import { TiledIconModule, TileParams } from './modules/tiledIconModule'
+
+import { DiscoverInsightsModule } from './modules/discoverInsight'
+import { Button, Layout, Space, Tooltip } from 'antd'
+import { SlackOutlined, UserAddOutlined, RocketOutlined, GithubOutlined } from '@ant-design/icons'
+
+import './home.scss'
+
+const { Content, Footer } = Layout
+import { useActions, useValues } from 'kea'
+
+import { teamLogic } from 'scenes/teamLogic'
+import { CreateInviteModalWithButton } from 'scenes/organization/Settings/CreateInviteModal'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { userLogic } from 'scenes/userLogic'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+
+function HeaderCTAs(): JSX.Element {
+    const { setInviteMembersModalOpen } = useActions(navigationLogic)
+    const { inviteMembersModalOpen } = useValues(navigationLogic)
+    const { preflight } = useValues(preflightLogic)
+    const [showToolTip, setShowToolTip] = useState(false)
+
+    const inviteModal = (
+        <div>
+            <Tooltip
+                placement="bottom"
+                visible={showToolTip && !inviteMembersModalOpen}
+                title={"Because insights are better when they're shared with friends."}
+            >
+                <div>
+                    {preflight?.email_service_available ? (
+                        <Button
+                            onMouseLeave={() => {
+                                setShowToolTip(false)
```

Comment:
Yes unfortunately that wasn't the behavior I was experiencing. I believe cause this tooltip is basically wrapping the entire modal, since it's wrapped around the button which contains the modal (just hidden) so when the modal is active it thinks its div is active too, hence the workarounds

Reply:
K, no point spending too much time on this right now